css/style.css

```diff
@@ -669,6 +667,20 @@ color: #344563;
     color: #fff;
   }
 
+  /* .form-error {
+    display: flex;
+    justify-content: center;
+    background-color: crimson;
+    color: #fff;
+  } */
+
+  
+  /* .error {
+    padding: 10px 20px;
+    margin: 10px auto;
+  } */
```

Comment:
- [ OPTIONAL] I think it would be better if you uncommented this styles and add them to the error message that's very important for user experince (UX).🔥🔥

Reply:
oh oh wow i thought i uncommented it before making the pull request. i will make surt to do that asap thabks